hosted/gke/p0/p0_suite_test.go

```diff
@@ -1,13 +1,34 @@
 package p0_test
 
 import (
+	"os"
 	"testing"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"github.com/rancher/hosted-providers-e2e/hosted/helpers"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	namegen "github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+)
+
+var (
+	ctx         helpers.Context
+	cluster     *management.Cluster
+	err         error
```

Comment:
I think defining these variables at a global level makes them more error prone, especially the `err` var.

Reply:
Took a cue from https://github.com/rancher/elemental/blob/main/tests/e2e/suite_test.go to declare suite level variables, however changed a few now to local